### PR TITLE
Adds support for the partition_strategy flag to ImportReadsCommand.

### PR DESCRIPTION
**Pull request by mbookman**
Updates dependency on google-api-services-genomics to the latest.
Updates CallVariantsCommand to adjust for read group set IDs parameter change
from list to singleton.

### CODE REVIEW

File: src/main/java/com/google/cloud/genomics/api/client/commands/CallVariantsCommand.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
 import com.google.api.services.genomics.model.Job;
 
 import java.io.IOException;
+import java.util.Arrays;
 import java.util.List;
 
 @Parameters(commandDescription = "Call variants on existing read group sets or" +
```
Comment (pgrosu) [inline src/main/java/com/google/cloud/genomics/api/client/commands/CallVariantsCommand.java]: But this says *on existing read group sets* which is plural, while below (on line 41-42) it is limited to one `readGroupSetId` which is singular.  Why are we shrinking the search space?

On another note it should be either calls or variants.  Maybe we could change it to return calls from a set of readgroups.

Comment (mbookman) [inline src/main/java/com/google/cloud/genomics/api/client/commands/CallVariantsCommand.java]: Thanks Paul.
Corrected to be singular "an existing read group set".
This command actually initiates variant calling, so I believe "Call variants ..." is the correct thing to say here. Executing the command would return a job id. It does not return calls or variants.

Comment (pgrosu) [inline src/main/java/com/google/cloud/genomics/api/client/commands/CallVariantsCommand.java]: Ah, yes the first word is always the verb - thanks for the help on the clarification.  Just out of curiosity, why am I getting the following result:

```
$ java -jar genomics-tools-client-java-v1beta2.jar callvariants --dataset_id 10473108253681171589 --read_group_set_id CMvnhpKTFhDq9e2Yy9G-Bg
Saving called variants into: 1000 Genomes
API request failed: Unknown field name: readGroupSetIds (400)
```

Thanks,
~p

**Comment by calbach** [comment]
Looks good. Thanks.

**Comment by pgrosu** [comment]
Thank you for the merge.  Also could the `genomics-tools-client-java-v1beta2.jar` be please updated, since I would like to re-run the following command in case the output changes:

`java -jar genomics-tools-client-java-v1beta2.jar callvariants --dataset_id 10473108253681171589 --read_group_set_id CMvnhpKTFhDq9e2Yy9G-Bg`

The `genomics-tools-client-java-v1beta.jar` file can probably be removed since `v1beta` is no longer supported in favor of `v1beta2`.

Thanks,
~p

**Pull request merged**